doc/source/examining/loading_data.rst

```diff
@@ -1961,10 +1962,25 @@ have assumed your data is stored in the three-dimensional array
 Stretched Grid Data
 -------------------
 
+.. warning::
+
+   API consistency for loading stretched grids is not guaranteed until yt 4.2!
```

Comment:
I'd suggest not making such version-specific promises. It'd be a shame to hold 4.2 just because of an issue we haven't anticipated yet.